### PR TITLE
Rename setTime method to setZipEntryTime

### PR DESCRIPTION
**Pull request by jorsol**
Rename the method to be more clear of the intent.

### CODE REVIEW

File: src/main/java/org/codehaus/plexus/archiver/zip/AbstractZipArchiver.java [hunk 1]
```diff
@@ -534,35 +536,28 @@ public InputStream get()
...
-         if (zipEntry.getExtra() != null){
-         // Uh-oh. What do we do now.
-         throw new IllegalStateException("DIdnt expect to see xtradata here ?");
-
-         }  else {
-         zipEntry.setExtra(ts.getLocalFileDataData());
-         }
-         */
+        // Round up 2 seconds when needed
+        lastModifiedTime = ( lastModifiedTime + 1999 ) - ( ( lastModifiedTime + 1999 ) % 2000 );
```
Comment (plamentotev) [inline src/main/java/org/codehaus/plexus/archiver/zip/AbstractZipArchiver.java:559]: Looks like this is change in how `setTime` works, not only method rename as the commit suggest, or I'm not reading the diff correctly?

Comment (jorsol) [reply src/main/java/org/codehaus/plexus/archiver/zip/AbstractZipArchiver.java:559]: I don't remember right now why I made this change, but it actually preserve the current behavior and it "normalizes" the time by truncating the millis.

Anyway, if you think that this is weird I could just simply remove it and leave it as before.

Comment (jorsol) [reply src/main/java/org/codehaus/plexus/archiver/zip/AbstractZipArchiver.java]: Reverted this line and also improved a bit the test `testLastModifiedTimeRounding`.

**Comment by jorsol** [comment]
Any other comment about this?

**Pull request merged**